js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
I can't reproduce this.

I do see that after the invalid coupon is applied, but before the new one is added, the error message isn't cleared. I've fixed that.